electrode-ota-server-dao-mariadb/src/queries/PackageQueries.ts

```diff
@@ -103,4 +103,6 @@ export const PackageQueries = {
                     WHERE id = ?`,
 
     updatePackageTime : `UPDATE package SET update_time = CURRENT_TIMESTAMP(3) WHERE id = ?`,
+
+    getMostRecentlyPromotedVersions : `SELECT app_version FROM package WHERE release_method IN ('Promote') ORDER BY update_time DESC LIMIT 3`,
```

Comment:
is this query going to be slow?

Reply:
Definitely not as the table has very lesser number(apporx 400) of data.

Reply:
You do know this is an ALL table scan.  Let's hope it never gets big.